recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,8 @@
+c_compiler:
+  - clang    # [ppc64le]
+c_compiler_version:
+  - 18       # [ppc64le]
```

Comment:
I saw that the version must be specified (https://github.com/conda-forge/libjxl-split-feedstock/pull/29/commits/475c28fa583dfd5e271b7d1c9fbd4bd717757974). I'm wondering if this could be automatically updated by conda-forge-admin.

Related: https://conda-forge.org/news/2024/04/30/clang-everywhere/

Reply:
The `conda_build_config.yaml` file is used to manually override or specify the compiler and its version. I followed the article you referred to (please see the summary) because the package failed to build with GCC.

Reply:
Also, could you add a comment with a link to related issues saying that we forced clang due to ICE in GCC?

Reply:
> The `conda_build_config.yaml` file is used to manually override or specify the compiler and its version. I followed the article you referred to (please see the summary) because the package failed to build with GCC.

I know, I'm just asking whether we have to manually upgrade the compiler version in the future to be compatible with the [pinned dependencies](https://conda-forge.org/docs/maintainer/pinning_deps/). It seems that the Clang toolchain hasn't been pinned yet and it might cause problems due to conflicting C++ std library versions.

Reply:
Hm, I'm not sure about pinning the Clang toolchain. Feel free to make any changes that make sense to you.